src/go/k8s/config/rbac/role.yaml

```diff
@@ -21,11 +21,34 @@ rules:
   - cert-manager.io
   resources:
   - certificates
+  verbs:
+  - create
```

Comment:
question: Is there a best practice of splitting the resources?
:thinking: since it seems like verbs per resource are the same.

Reply:
these files are generated. I don't follow the question though, sorry. Can you try to phrase it differently :-)

Reply:
@0xdiba the generator generally split them even though in some cases (like here) they have the same verbs.

Reply:
This can be addressed by the changing the `cluster_controller.go`. When I change the:

```
//+kubebuilder:rbac:groups=core,resources=services,verbs=get;list;watch;create;update;patch;
//+kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;create;update;patch;
//+kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;watch;
//+kubebuilder:rbac:groups=core,resources=secrets,verbs=get;list;watch;
//+kubebuilder:rbac:groups=cert-manager.io,resources=issuers,verbs=create;get;list;watch;patch;delete;
//+kubebuilder:rbac:groups=cert-manager.io,resources=certificates,verbs=create;get;list;watch;patch;delete;
//+kubebuilder:rbac:groups=cert-manager.io,resources=clusterissuers,verbs=create;get;list;watch;patch;delete;
```

to

```
//+kubebuilder:rbac:groups=core,resources=services;configmaps,verbs=get;list;watch;create;update;patch;
//+kubebuilder:rbac:groups=core,resources=pods;secrets,verbs=get;list;watch;
//+kubebuilder:rbac:groups=cert-manager.io,resources=issuers;certificates;clusterissuers,verbs=create;get;list;watch;patch;delete;
```

then we have smaller set of yaml